examples/flash_attention/example_gqa_fwd_bshd.py

```
@@ -268,6 +268,33 @@ def main(batch: int = 1,
         print(f"Ref latency: {ref_latency}")
 
 
+def benchmark(batch: int = 1,
+              heads: int = 64,
+              seq_len: int = 4096,
+              dim: int = 128,
+              is_causal: bool = False,
+              groups: int = 16,
+              tune: bool = False):
```

Ruff (0.14.8)

Unused function argument: `tune`

(ARG001)